src/main/java/cms/gov/madie/measure/services/MeasureService.java

```diff
@@ -489,6 +489,36 @@ public Map<String, List<AclSpecification>> shareMeasures(
       Map<String, List<String>> measureUserIdMap, String username) {
     Map<String, List<AclSpecification>> measureIdToAclSpecification = new HashMap<>();
 
+    verifyShareAuthorization(measureUserIdMap, username);
+
+    measureUserIdMap.forEach(
+        (measureId, userIds) -> {
+          AclOperation aclOperation = buildShareAclOperation(userIds);
+          measureIdToAclSpecification.put(
+              measureId, updateAccessControlList(measureId, aclOperation, username));
+        });
+
+    return measureIdToAclSpecification;
```

Comment:
Do you think we can add any logs that will help us in debugging in future ?

Reply:
We may not need any logs in your recent work, but it is would be nice to have few logs when there are exceptions